beamline.rst

```diff
@@ -1,212 +1,379 @@
-Beamline API
```

Comment:
I would avoid General here, we dont have to use beamline but the intention with this was to define functions that
are "beamline wide".